urdf/ur.urdf.xacro

```diff
@@ -5,12 +5,15 @@
    <!-- import main macro -->
    <xacro:include filename="$(find ur_description)/urdf/ur_macro.xacro"/>
 
+   <!-- possible 'ur_type' values: ur3, ur3e, ur5, ur5e, ur10, ur10e, ur16e -->
+   <xacro:arg name="ur_type" default=""/>
```

Comment:
Making this variable required (by not giving it a default value) might be a good idea, as otherwise we might end up with non-existing parameter file paths if this is skipped.

Reply:
```suggestion
   <xacro:arg name="ur_type"/>
```